event_types.md

```diff
@@ -246,7 +241,318 @@ connection: keep-alive
 }
 ```
 
+## Daily Statistics Complete Event
+
+Triggered when the processing/ingestion of daily statistics aggregates has completed for a given day.
+
+**Webhook event name:**
+
+`daily_statistics_complete`
+
+**Webhook Payload Parameters:**
+
+|Name|Type|Description|
+|:---|:---|:----------|
+|id|Number|Daily Statistics Record ID|
+|studyId|Number|CentrePoint Study ID|
+|siteId|Number|CentrePoint Site ID|
+|subjectId|Number|CentrePoint Subject ID|
+|date|String (ISO8601 Date/Time)|Date of daily statistics record in subject's timezone|
+|activityMonitorSerials|Array of String|Activity monitor serial number(s) which contributed data to the daily statistic record|
+|epochAggregation|Object|Contains daily summary of epoch data|
+|freedsonAggregations|Object|Contains daily summary of Freedson cutpoints|
+|mavmAggregation|Object|Contains daily summary of MAVMv1 steps|
+|crouterAggregations|Object|Contains daily summary of Crouter-Child cutpoints|
+|staudenmayerAggregations|Object|Contains daily summary of Staudenmayer cutpoints|
+|uwfAggregations|Object|Contains daily summary of UWF steps|
+
+### Epoch Aggregation
+
+|Name|Type|Description|
+|:---|:---|:----------|
+|wearMinutes|Number|A daily aggregate, in minutes, of the non-partial epochs for subject that represent when subject was wearing device based on wear detection algorithm|
+|nonWearMinutes|Number|A daily aggregate, in minutes, of the non-partial epochs for subject that represent when subject was not wearing device based on wear detection algorithm|
+|sleepMinutes|Number|A daily aggregate of the non-partial epochs for subject that represent when the subject was asleep
+ based on sleep detection algorithm|
```

Comment:
This new line is causing a new row in the table

Reply:
Corrected